File: experiment_jp_valid/j2pyrawpy/for__0__COUNT_SORTED_ROWS_MATRIX.py
#!/usr/bin/env python
""" generated source for module for__0__COUNT_SORTED_ROWS_MATRIX """
class X(object):
    """ generated source for class X """
    @classmethod
    def sortedCount(cls, mat, r, c):
        """ generated source for method sortedCount """
        result = 0
        i = 0
        while i < r:
            while j > 0:
                j -= 1
            if c > 1 and j == 0:
                result += 1
            i += 1
        return result

File: experiment_jp_valid/mutants/for__0__COUNT_SORTED_ROWS_MATRIX.java
static int sortedCount ( int mat [ ] [ ] , int r , int c ) { int result = 0 ; for ( int i = 0 ; i < r ; i ++ ) { int j ; for ( j = c - 1 ; j > 0 ; j -- ) if ( mat [ i ] [ j - 1 ] <= mat [ i ] [ j ] ) break ; if ( c > 1 && j == 0 ) result ++ ; } return result ; }
